README.md

```diff
@@ -108,6 +109,12 @@ Following these steps you will be able to run separately the **lookout** depende
     $ lookoutd serve --dry-run
     ```
 
+## Running a Distributed lookout
```

Comment:
maybe in distributed mode? but I'm fine with current word header too.

Reply:
`Running in Distributed Mode` would be slightly better, even if it's only because capitalization here looks incorrect (even if it is correct because it's using English heading capitalization + lookout in lowercase as it is in branding, but looks inconsistent).

Reply:
:+1: Change made; squashed and rebased.